src/CarroMessenger.php

```diff
@@ -58,7 +57,7 @@ public function sendWhatsAppTemplateSMSViaMsgBird($data)
             Log::error('WhatsApp template message failed @'. __FUNCTION__.' of '. __CLASS__,
             [$e->getMessage()]);
             
-            event(new MessageWasSent($this->model, new MessageFailedResponse()));
+            event(new MessageWasSent($model, new MessageFailedResponse()));
```

Comment:
```suggestion
            event(new MessageWasSent($model, new MessageFailedResponse()));
```
What if model is null here bro?

Reply:
Model won't be null bro..we always pass that model(communication)